features/bootstrap/ApplicationContext.php

```diff
@@ -120,28 +120,42 @@ public function iRunPhpspec($formatter = null, $option = null, $interactive = nu
      */
     public function iRunPhpspecAndAnswerWhenAskedIfIWantToGenerateTheCode($answer, $option=null)
     {
-        $arguments = array (
-            'command' => 'run'
-        );
-
-        $this->addOptionToArguments($option, $arguments);
-
-        $this->prompter->setAnswer($answer=='y');
-
-        $this->lastExitCode = $this->tester->run($arguments, array('interactive' => true));
+        $this->runPhpSpecAndAnswerQuestions($answer, 1, $option);
     }
 
     /**
      * @When I run phpspec and answer :answer to both questions
      */
     public function iRunPhpspecAndAnswerToBothQuestions($answer)
     {
+        $this->runPhpSpecAndAnswerQuestions($answer, 2);
```

Comment:
I suggest combining this with your new one and transforming `both` / `three` into ints